helix-term/src/commands.rs

```diff
@@ -1062,6 +1090,20 @@ mod cmd {
             fun: format,
             completer: None,
         },
+        Command {
+            name: "earlier",
+            alias: None,
+            doc: "Jump back to an earlier point in edit history.",
+            fun: earlier,
+            completer: None,
+        },
+        Command {
+            name: "later",
+            alias: None,
```

Comment:
```suggestion
            alias: Some("lat"),
```
Maybe we can follow vim here.